public/js/circos.js

```diff
@@ -25,6 +25,13 @@ class Graph {
 
     constructor($svgContainer, props) {
         this.queries = props.queries;
+        this.query_arr = [];
+        _.each(this.queries, _.bind(function (obj) {
```

Comment:
Can you make some more simplifications here please. For example, you can directly iterate over `props.queries` instead of first assigning it to `this.queries`.